docs/source/pagination/directional-pagers.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: Directional Pagination
+---
+
+`GraphQLQueryPager` supports pagination in both the forward and reverse direction, as well as both at once. 
+
+## Forward Pagination
+
+Forward pagination is the most common form of pagination. It is used to fetch the next `n` items in a list. We can use the convenience `make` functions to create a configured `GraphQLQueryPager`. While we have many options depending on our requirements -- whether we use one query or two, whether we want to use a cursor or an offset, whether we want to transform the results, etc. -- we will examine using a cursor with a single query.
```

Comment:
Let's talk about my question related to omitting `paginationDirection` in single directional convenience functions. If we can go that route, then I have some suggestions to tweak this.